src/qibosoq/rfsoc_server.py

```diff
@@ -66,16 +70,28 @@ def execute_program(self, data: dict) -> dict:
         qick_logger.handlers[0].doRollover()
         qick_logger.info(program.asm())
 
-        toti, totq = program.acquire(
-            self.server.qick_soc,
-            data["readouts_per_experiment"],
-            load_pulses=True,
-            progress=False,
-            debug=False,
-            average=data["average"],
-        )
-
-        return {"i": toti.tolist(), "q": totq.tolist()}
+        if opcode is OperationCode.EXECUTE_PULSE_SEQUENCE_RAW:
+            results = program.acquire_decimated(
+                self.server.qick_soc,
+                load_pulses=True,
+                progress=False,
+                debug=False,
+            )
+            toti = [[results[0][0].tolist()]]
```

Comment:
The raw data can be quite large. We have to check the TCP transmissions to give no problems with large amounts of data.

Reply:
For what I saw, the server automatically splits large packets in smaller ones so this *shouldn't* be a problem